matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/model/ChunkEntity.kt

```diff
@@ -42,4 +44,13 @@ internal open class ChunkEntity(@Index var prevToken: String? = null,
     val room: RealmResults<RoomEntity>? = null
 
     companion object
+
+    fun deleteOnCascade(deleteStateEvents: Boolean, canDeleteRoot: Boolean) {
```

Comment:
Can we keep the Entity files clean of transactions?

Reply:
Yes, but in this case, it's nice to see what is in the object (in the constructor) when you want to delete everything.
So having all in the same file is nice.
Otherwise we may in the future introduce new bug like that...
I can move it outside the class (as an extension) if you prefer, but I want to have all in the same file.

Reply:
As u want, we already made extensions for query. If u prefer, let them on the entity files